docs/containerignore.5.md

```diff
@@ -0,0 +1,87 @@
+% "containerignore" "28" "Sep 2021" "" "Container User Manuals"
+
+# NAME
+
+.containerignore(.dockerignore) - files to ignore buildah or podman build context directory
+
+# INTRODUCTION
+
+Before container engines build image, they look for a file named .containerignore or .dockerignore in the root
+directory of the context. If one of these file exists, the CLI modifies the context to exclude files and
```

Comment:
As we call it "context directory" elsewhere, I'd suggest continuing that here too.  I won't thumb wrestle about it though.

```suggestion
context directory. If one of these file exists, the CLI modifies the context to exclude files and
```